core-web/libs/sdk/uve/src/lib/utils.ts

```diff
@@ -57,3 +59,45 @@ export function getUVEState(): UVEState | undefined {
         publishDate
     };
 }
+
+/**
+ * Creates a subscription to a UVE event.
+ *
+ * @param {string} event - The event to subscribe to.
+ * @param {UVECallback} callback - The callback to call when the event is triggered.
+ * @return {UnsubscribeUVE | undefined} The unsubscribe function if the event is valid, undefined otherwise.
+ *
+ * @example
+ * ```ts
+ * const unsubscribeChanges = createUVESubscription('changes', (payload) => {
+ *   console.log(payload);
+ * });
+ * ```
+ */
+export function createUVESubscription(event: string, callback: UVECallback): UVESubscription {
+    if (!getUVEState()) {
+        console.warn('UVE Subscription: Not running inside UVE');
+
+        return {
+            unsubscribe: () => {
+                /* */
+            },
+            event
+        };
+    }
```

Comment:
```suggestion
    if (!getUVEState()) {
        console.warn('UVE Subscription: Not running inside UVE');

        return
    }
```

No?

Reply:
I want to keep the model, to prevent ambiguity. But we can discuss that further

Reply:
I'm throwing errors to prevent silent fails